libsolidity/inlineasm/AsmAnalysis.h

```diff
@@ -63,10 +63,10 @@ class AsmAnalyzer: public boost::static_visitor<bool>
 public:
 	explicit AsmAnalyzer(
 		AsmAnalysisInfo& _analysisInfo,
-		ErrorList& _errors,
+		ErrorReporter& _errorReporter,
```

Comment:
I guess this file should also have a forward declaration and the `Exceptions.h` include removed?